### PR TITLE
Modify git-dumper recipe

### PR DESCRIPTION
**Pull request by ph20**
Modify git-dumper recipe for work with last changes

### CODE REVIEW

**Review by noraj — APPROVED** [review]
LGTM

```
$ ba-dev -e 'pacman -Ql git-dumper' -p git-dumper-56.32d47a2-2-any.pkg.tar.zst
...
git-dumper /usr/
git-dumper /usr/bin/
git-dumper /usr/bin/git-dumper
git-dumper /usr/lib/
git-dumper /usr/lib/python3.10/
git-dumper /usr/lib/python3.10/site-packages/
git-dumper /usr/lib/python3.10/site-packages/git_dumper-1.0.6.dist-info/
git-dumper /usr/lib/python3.10/site-packages/git_dumper-1.0.6.dist-info/INSTALLER
git-dumper /usr/lib/python3.10/site-packages/git_dumper-1.0.6.dist-info/LICENSE
git-dumper /usr/lib/python3.10/site-packages/git_dumper-1.0.6.dist-info/METADATA
git-dumper /usr/lib/python3.10/site-packages/git_dumper-1.0.6.dist-info/RECORD
git-dumper /usr/lib/python3.10/site-packages/git_dumper-1.0.6.dist-info/REQUESTED
git-dumper /usr/lib/python3.10/site-packages/git_dumper-1.0.6.dist-info/WHEEL
git-dumper /usr/lib/python3.10/site-packages/git_dumper-1.0.6.dist-info/entry_points.txt
git-dumper /usr/lib/python3.10/site-packages/git_dumper-1.0.6.dist-info/top_level.txt
git-dumper /usr/lib/python3.10/site-packages/git_dumper.py
git-dumper /usr/share/
git-dumper /usr/share/licenses/
git-dumper /usr/share/licenses/git-dumper/
git-dumper /usr/share/licenses/git-dumper/LICENSE
```

File: packages/git-dumper/PKGBUILD [hunk 1]
```diff
@@ -3,35 +3,41 @@
...
          'python-requests')
-makedepends=('git' 'python-setuptools')
+makedepends=('git' 'python-build' 'python-pip')
 source=("git+https://github.com/arthaud/$pkgname.git")
 sha512sums=('SKIP')
 
 pkgver() {
   cd $pkgname
-
   echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
```
Comment (noraj) [inline packages/git-dumper/PKGBUILD:20]: `git describe --long --tags | sed 's/\([^-]*-g\)/r\1/;s/-/./g'`

File: packages/git-dumper/PKGBUILD [hunk 2]
```diff
@@ -3,35 +3,41 @@
 
 pkgname=git-dumper
 pkgver=56.32d47a2
```
Comment (noraj) [inline packages/git-dumper/PKGBUILD:5]: `pkgver=1.0.6.r0.g32d47a2`

**Review by noraj — CHANGES_REQUESTED** [review]

File: packages/git-dumper/PKGBUILD (hunk 2 above)
Comment (ph20) [reply packages/git-dumper/PKGBUILD]: Thanks corrected

File: packages/git-dumper/PKGBUILD (hunk 1 above)
Comment (ph20) [reply packages/git-dumper/PKGBUILD]: Thanks, corrected

**Pull request merged**